htdocs/openwebrx.js

```diff
@@ -73,7 +73,10 @@ var waterfall_min_level_default;
 var waterfall_max_level_default;
 var waterfall_colors = buildWaterfallColors(['#000', '#FFF']);
 var waterfall_auto_levels;
+var waterfall_auto_level_default_mode;
 var waterfall_auto_min_range;
+var waterfall_measure_minmax_now = false;
+var waterfall_measure_minmax_continuous = false;
 
```

Comment:
It looks as if `waterfall_auto_level_default_mode` is never really read, so I'd suggest removing it.